### ISSUE TITLE
BERT NER very slow across multiple docker containers

### ISSUE THREAD

**Issue by tnmcneil** [kamalkraj/BERT-NER#32]
When running BERT NER on a single container the code executes fine, but when scaled to run across 2+ containers the speed drops drastically (from 20 seconds -> 20 minutes)
This has been isolated as definitely being a problem with BERT as it does not happen without it,  wondering if you have come across anything like this 
The problem specifically comes about when calling model.predict()
for one sentence it takes ~20s to output a prediction
EDIT:
it is slowing down at lines 88-89 in bert.py:
` with torch.no_grad():`
`logits = self.model(input_ids, segment_ids, input_mask,valid_ids) `

**Comment by kamalkraj** [comment]
Hi @tnmcneil,
Check the memory usage, if you're running all the containers on a single machine. 
Line 88-89 is the model prediction.

**Comment by tnmcneil** [comment]
@kamalkraj memory usage looks fine but CPU usage is at 100% (and scales up with how many cores are on the machine)

I believe it may be linked to this issue: https://github.com/google-research/bert/issues/753#issue-465847403
